tests/unit/models/test_model_interfaces.py

```diff
@@ -651,14 +673,70 @@ def test_sparse_variational_optimize(batcher, compile: bool) -> None:
 
     optimizer = create_optimizer(
         tf.optimizers.Adam(),
-        dict(max_iter=20, batch_size=10, dataset_builder=batcher, compile=compile),
+        dict(max_iter=10, batch_size=10, dataset_builder=batcher, compile=compile),
     )
     model = SparseVariational(_svgp(x_observed[:10]), dataset, optimizer=optimizer)
     loss = model.model.training_loss(data)
     model.optimize(dataset)
     assert model.model.training_loss(data) < loss
 
 
+@pytest.mark.parametrize("use_natgrads", [True, False])
+@pytest.mark.parametrize("compile", [True, False])
```

Comment:
I would move `compile` argument to `conftest.py` file as a fixture, then it doesn't need to be parametrized above the testing function - it seems like an argument that would be tested in several places at least

Reply:
There is no conftest.py

Reply:
there is one at `tests/conftest.py`, thats the main one, although you can make it subfolder specific as well, by simply creating another one at the subfolder level